pyproject.toml

```diff
@@ -125,6 +125,7 @@ dependencies = [
     "types-urllib3",
     "types-jinja2",
     "types-babel",
+    "aiohttp==3.9.0b1; python_version >= '3.12'", # pre-release supports py3.12
```

Comment:
Seems that `aiohttp==3.9.0` has been released on Nov 18:

 * https://pypi.org/project/aiohttp/3.9.0/

So, I guess, this is not needed any more?